core/domain/exp_services.py

```diff
@@ -665,6 +665,7 @@ def save_new_exploration(committer_id, exploration):
     }])
     user_services.add_created_exploration_id(committer_id, exploration.id)
     user_services.add_edited_exploration_id(committer_id, exploration.id)
+    user_services.record_user_created_an_exploration(committer_id)
```

Comment:
Hi @seanlip, when exploration is created it will record only last_created_en_exploration. According to you last_edited_an_exploration should be changed even for creating an exploration, right ?
That's what I forgot here and seems like you missed this too.